spec.html

```diff
@@ -16175,16 +16175,16 @@
               for (let key of Reflect.ownKeys(obj)) {
```

Comment:
It is mostly a matter of code style. Some might argue that `const` is for declarations that will never ever be reassigned, like number constants or `require`s. And `let`s are for any other declarations: this keeps diffs cleaner.

Lets make them `const`s to reflect how most people write ES201* these days.